src/Nethermind/Nethermind.Network.Discovery/Lifecycle/NodeLifecycleManager.cs

```diff
@@ -370,6 +372,7 @@ private void UpdateState(NodeLifecycleState newState)
 
         State = newState;
         OnStateChanged?.Invoke(this, State);
+        if (newState == NodeLifecycleState.Active && string.IsNullOrEmpty(ManagedNode.Enr)) SendEnrRequest();
```

Comment:
We don't practice one line ifs, because code surrounded by curly braces looks safer